app/controllers/v0/sign_in_controller.rb

```diff
@@ -181,25 +184,26 @@ def logout # rubocop:disable Metrics/MethodLength
         raise SignIn::Errors::MalformedParamsError.new message: 'Client id is not valid'
       end
 
-      unless load_user(skip_expiration_check: true)
-        raise SignIn::Errors::LogoutAuthorizationError.new message: 'Unable to Authorize User'
+      unless access_token_authenticate(skip_error_handling: true)
+        raise SignIn::Errors::LogoutAuthorizationError.new message: 'Unable to authorize access token'
       end
 
+      session = SignIn::OAuthSession.find_by(handle: @access_token&.session_handle)
```

Comment:
I think you can assume `@access_token` exists here. Otherwise the `LogoutAuthorizationError` would have been raised above

Reply:
(so you can remove the `&.` safety operator)